### ISSUE TITLE
Agents as Dynamic Obstacles

### ISSUE THREAD

**Issue by nsfw** [sybrenstuvel/Python-RVO2#4]
Howdy! 
I'm brand new to this RVO2 library, but I'm wondering if you could answer a question, namely:

I have some Agents that need to animate along a predetermined animation path, that is, the can't participate in the "reciprocal" part of RVO; for the other Agents, these are basically Dynamic Obstacles.

Is there some way to set the Agent parameters to make something like this happen? I'm considering turning down the time horizon for the "obstacles" and maybe turning it up for the "avoiders", or just resetting the obstacle agent's position each step.

I thought that RVO supported setting a weight, i.e. 0.5 which had the effect of having each agent get out of the way of the other with equal effort. I was hoping I could find a place to set the weight to 1.0 for my dynamic obstacle objects. 

I realize that this is just the home of the Python bindings for the RVO2 library - but I figure you're also expert users!

Thanks in advance

**Comment by sybrenstuvel** [comment]
As you said, this repo is just for the Python bindings. I haven't used RVO2 much, mostly just to compare with my own crowd simulation technique [Torso Crowds](https://stuvel.eu/publications).

The C++ implementation isn't that hard to follow. There are a few mentions of `0.5f`, so maybe those are a starting point for you to look at?

**Comment by sybrenstuvel** [comment]
Closing due to inactivity.
